.github/workflows/protocol-devchain-anvil.yml

```diff
@@ -0,0 +1,129 @@
+name: Protocol Devchain Anvil
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
Just for testing purposes, once successfully published it will run just on push to master

Reply:
removed